zellij-utils/src/cli.rs

```diff
@@ -85,6 +85,11 @@ pub enum Sessions {
         /// zellij client (if any) and attach to this.
         #[structopt(long, short)]
         force: bool,
+
+        /// Create a session if one does not exist.
+        #[structopt(long)]
```

Comment:
Let's have the `short` version as well.

Reply:
Of course
I initially removed the short option because it was for `zellij` and not for `attach`
Added